src/plugins/github/graphql.js

```diff
@@ -885,6 +983,32 @@ function commitHistoryFragment(): FragmentDefinition {
   ]);
 }
 
+export type ReactionContent =
+  | "THUMBS_UP"
+  | "THUMBS_DOWN"
+  | "LAUGH"
+  | "CONFUSED"
+  | "HEART"
+  | "HOORAY";
+
+export type ReactionJSON = {|
+  +id: string,
+  +content: ReactionContent,
+  +user: AuthorJSON,
```

Comment:
Please mark that this is nullable, even though we do not know why.

(If a user deletes their account, their reactions are removed.)